data/brands/amenity/fast_food.json

```
@@ -1438,6 +1438,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Goli Vada Pav",
+      "id": "golivadapav-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "alt_name": "Goli Vadapav No.1",
+        "amenity": "fast_food",
+        "brand": "Goli Vada Pav Pvt. Ltd",
+        "brand:wikidata": "Q5580697",
+        "brand:wikipedia": "en:Goli_Vada_Pav",
```

Need to remove underscores when adding the wiki link..
```suggestion
        "brand:wikipedia": "en:Goli Vada Pav",
```